src/Microsoft.AspNet.Razor/Parser/TagHelpers/TagHelperParseTreeRewriter.cs

```diff
@@ -70,7 +70,7 @@ private void RewriteTags(Block input, RewritingContext context)
                 // At this point the child is a Span or Block with Type BlockType.Tag that doesn't happen to be a
                 // tag helper.
 
-                // Add the child to current block. 
+                // Add the child to current block.
```

Comment:
Not going to bother undoing. Having the cleanup trailing whitespace addon auto-removes these bits. More work not doing this then just letting it happen.